### PR TITLE
social: Mark mastodon account with rel="me"

### PR DESCRIPTION
**Pull request by chrysn**
This denotes that that account is run by the same people as this site
using [1], and will turn the "web" link there to green (indicating to
the user that fosstodon.org has verified that this account is endorsed
by riot-os.org).

[1]: https://indieweb.org/rel-me

---

This is in draft mode while I'm getting familiar with how the CI build previews work; I'll switch it to ready-for-review when i think that the fosstodon integrations are ready for prime time.

### CODE REVIEW

**Comment by chrysn** [comment]
Will look like this, then (with the web site temporarily switched over to the PR preview):

screenshot

**Comment by chrysn** [comment]
Now that the account is on (see also <https://forum.riot-os.org/t/re-riot-maintainer-twitter-account-mastodon-presence/3228/2>), this is ready IMO.

**Comment by leandrolanzieri** [comment]
The community page needs updating with the change of the handle https://github.com/RIOT-OS/riot-os.org/blob/master/community.html (including the image). Would you update that on this PR @chrysn or should I give it a try?

**Comment by chrysn** [comment]
On the text side I updated it, but on the image side I'm stuck as to where to find the preferred form of modification: the handle text is part of the picture whose source I didn't find.

Can't do much more today, so viable options for me are:
* merge as is (and do one of the below later),
* you fix the image,
* I get pointers to the image sources and update them (but can be some time, me being on vacation and having very unpredictable times).

**Comment by leandrolanzieri** [comment]
Maybe @waehlisch has the sources?

**Comment by waehlisch** [comment]
which images are we talking about?

**Comment by leandrolanzieri** [comment]
> which images are we talking about?

This https://github.com/RIOT-OS/riot-os.org/blob/master/assets/img/mastodon-logo-riot.png

File: community.html
Comment (leandrolanzieri) [inline community.html:95]: BTW, the button "Our news" also needs to be updated to the new URL.

**Comment by waehlisch** [comment]
mastodon-logo-riot

**Comment by chrysn** [comment]
Thanks, fixed; waiting for CI to preview.

If it looks right, can I rebase to squash squash the fixups?

**Comment by leandrolanzieri** [comment]
> If it looks right [edit: it does], can I rebase to squash squash the fixups?

Please go ahead

**Comment by chrysn** [comment]
Thanks, rebased into logical commits.

**Review by leandrolanzieri — APPROVED** [review]
Changes look good, ACK!

**Comment by leandrolanzieri** [comment]
Thanks @chrysn !

**Pull request merged**
